charts/tfy-k8s-azure-aks-inframold/templates/prometheus-config.yaml

```diff
@@ -30,6 +30,9 @@ spec:
         {{- if .Values.aws }}
           karpenter:
             enabled: {{ .Values.aws.karpenter.enabled }}
+        {{- else }}
+          karpenter:
```

Comment:
why would you need a else section?

Reply:
The else section is for other clouds not AWS. This was causing issues with the helm template.